### PR TITLE
E2E - use the new method gotoMap in some tests

### PR DESCRIPTION
**Pull request by Gustry**

### CODE REVIEW

File: tests/end2end/playwright/snap.spec.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // @ts-check
 import { test, expect } from '@playwright/test';
+const { gotoMap } = require('./globals')
```
Comment (rldhont) [inline tests/end2end/playwright/snap.spec.js:3]:
```suggestion
import { gotoMap } from './globals';
```

File: tests/end2end/playwright/treeview.spec.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // @ts-check
 import { test, expect } from '@playwright/test';
+const { gotoMap } = require('./globals')
```
Comment (rldhont) [inline tests/end2end/playwright/treeview.spec.js:3]:
```suggestion
import { gotoMap } from './globals';
```

File: tests/end2end/playwright/webdav-upload.spec.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
 // @ts-check
 import { test, expect } from '@playwright/test';
+const { gotoMap } = require('./globals')
```
Comment (rldhont) [inline tests/end2end/playwright/webdav-upload.spec.js:3]:
```suggestion
import { gotoMap } from './globals';
```

**Review by rldhont — CHANGES_REQUESTED** [review]
Prefer `import ... from` instead of `require`

**Comment by Gustry** [comment]
One suggestion is enough @rldhont ;-)

I will close and open again after #4601 and #4602

**Pull request closed without merging**